docs/source/guides/VTFModding.rst

```diff
@@ -38,35 +86,81 @@ Though it is more a skins thing, making vtf skins is somewhat complicated, and t
+
+VTFEdit
+--------
+
+`VTFEdit`_ is a tool to edit, view, and create .vtf files.
+
+.. _VTFEdit: https://nemstools.github.io/pages/VTFLib-Download.html
+
+Launch `VTFEdit`_. Top left, click ``File``, ``Import``, find and Import your custom texture(s). 
+
+When importing your normal map. Choose to import as a ``Volume Map``
+When importing your diffuse map. Choose to import as a ``Animated Map``
+
+More info about .vtf format possibilities `here <https://retryy.gitbook.io/tf2/wiki/create/formats>`_, or the official source docs `here <https://developer.valvesoftware.com/wiki/Valve_Texture_Format>`_.
+
+After that, save your new .vtf's into the same folder as your custom .vmt with a simple name.
+
+Configuring your .vmt
+---------------------
+
+To know which texture/texture map you have. Analyze `this <https://retryy.gitbook.io/tf2/wiki/create/texturemaps>`_ wiki.
+
+In the ``"$basetexture"`` argument. You'll enter your .vtf texture directory. We'll use ``models\weapons_r2\car_ctm\NAMEOFVTF``. This is pointing to your custom diffuse .vtf with the simple name. Do not add the ``.vtf`` file extension to the end of the directory; ``models\weapons_r2\car_ctm\NAMEOFVTF.vtf``, or the game cannot find the texture.
+
+Do the same for your normal map in the ``"$bumpmap"`` argument.
+
+Now create another vtf with literally any image. Put its path in ``"$texture2"`` argument. As far as i know, this is necessary even though the texture isn't used. (i never had to do this but maybe you need it)
+
+Final VPK folder
+----------------
+
+Your root folder should look somewhat like this
+
+::
 
 	root
 	├─ materials
 	│  └─ models
 	│     └─ weapons_r2
-	│        └─ vtfkraber
-	│           └─ vtfkraber.vmt
+	│        └─ car_smg
+	│           ├─ YOURTEXTURE.vtf
+	│           ├─ YOURTEXTURE.vtf
+	│           └─ car_ctm.vmt
 	└─ models
 	   └─ weapons
-	      └─at_rifle (name of kraber)
-	        ├─ ptpov_at_rifle.mdl
-	        └─ w_at_rifle.mdl
+	      └─car101
+	        ├─ ptpov_car101.mdl
+	        └─ w_car101.mdl
 
-And you're done! You just need to pack it into a vpk with the vpk tool and put it in a mod.
+Finished.
+---------
+
+You're done! You just need to pack it into a vpk with the vpk tool (for our gun mod, we'd repack to ``englishclient_mp_common.pak000_dir.vpk``), and put it into a northstar mod. 
+
+Help with repacking `here <https://noskill.gitbook.io/titanfall2/intro/duction/vpk-packpack>`_, and help with Northstar mods `here <https://r2northstar.readthedocs.io/en/latest/guides/gettingstarted.html>`_.
 
 Making your Skin Animated
 -------------------------
 
-To make an animated skin, all we need to do is add a proxie and change our albedo vtf. Once you've made the frames of your skin, import them all at once with ctrl+a and save your vtf. Put it as `"$basecolor"`. At the bottom of your vmt but before the }, add this:
+To add animation functionality, all we need to do is add a proxie and change our albedo vtf. 
```

Comment:
explain what a proxie is and how/where we add one.

Reply:
I don't know what they are. So I will learn and then add that.